### ISSUE TITLE
Planning and reinforcement learning.

### ISSUE THREAD

**Issue by viking-sudo-rm** [viking-sudo-rm/StackNN#11]
Investigate using an RL training objective and incorporating planning.

@viking-sudo-rm might look into this for Advanced NLP final project.

**Comment by viking-sudo-rm** [comment]
Lots of discussion during CLAY meetings.

Exploring model-based and policy-based methods for utilizing data structures in a neural network.

**Comment by viking-sudo-rm** [comment]
See https://github.com/viking-sudo-rm/policy-gradients for our policy-gradient reverse implementation.
